loops-arrays-objects.js

```diff
@@ -145,24 +188,18 @@ console.log(res);
 
```

Comment:
Volgens mij is het niet de bedoeling om de  NovieEmployees array te wijzigen en niet een nieuwe res te maken, verder prima.